modules/system/nvram.nix

```diff
@@ -0,0 +1,36 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.system;
+
+  mkNvramVariables =
+    lib.attrsets.mapAttrsToList
+      (name: value: "nvram ${name}=${value}")
+      cfg.nvram.variables;
+in
+
+{
+  meta.maintainers = [
+    lib.maintainers.samasaur or "samasaur"
+  ];
+
+  options = {
+    system.nvram.variables = lib.mkOption {
+      type = with lib.types; attrsOf str;
+      default = {};
+      internal = true;
+      example = ''
+        # produces `nvram StartupMute=%01` (or `%00`)
+        system.nvram."StartupMute" = if chime then "%00" else "%01";
+      '';
```

Comment:
This should be in `description`, whereas `example` is for values like `example = { StartupMute = "%00"; }`

Reply:
whoops, my bad